examples/plot_logistic_tick_vs_scikit.py

```diff
@@ -0,0 +1,84 @@
+"""
+================================================================
+Logistic regression comparison: ``scikit-learn`` versus ``tick``
+================================================================
+
+In this example we give a naive comparison of ``tick`` and ``scikit-learn`` for
+binary classification using logistic regression with :math:`\ell_1`
+penalization.
+This comparison is done using the well-known ``adult`` dataset, a standard
+benchmark dataset for binary clasification.
+Some remarks are the following:
+
+* Both classifiers have the same performance in terms of AUC (area under the
+  ROC curve)
+* Learned model-weights are slightly different. This is explained by the fact
+  that ``scikit-learn`` uses ``liblinear`` for optimization of the
+  :math:`\ell_1`-penalized likelihood. When using this solver, the
+  ``intercept`` is penalized like the model weights (``coeff_``), while this is
+  not the case in `tick`. Note that this difference can be reduced by tuning the
+  ``intercept_scaling`` parameter from ``scikit-learn``'s
+  ``LogisticRegression``
+* In this example, the computational time of ``tick`` is better than ``scikit``'s
+"""
+import numpy as np
+from time import time
+import matplotlib.pyplot as plt
+
+from sklearn.metrics import roc_curve, auc
+from sklearn.linear_model import LogisticRegression as LogRegScikit
+
+from tick.dataset import fetch_tick_dataset
+from tick.inference import LogisticRegression as LogRegTick
+
+train_set = fetch_tick_dataset('binary/adult/adult.trn.bz2')
+test_set = fetch_tick_dataset('binary/adult/adult.tst.bz2')
+
+clf_tick = LogRegTick(C=1e5, penalty='l1')
+clf_scikit = LogRegScikit(penalty='l1')
+
+t1 = time()
+clf_tick.fit(train_set[0], train_set[1])
+t_tick = time() - t1
```

Comment:
for performance measurement it's better to use time.clock()
http://pythoncentral.io/measure-time-in-python-time-time-vs-time-clock/